options/locale/locale_en-US.ini

```diff
@@ -116,15 +116,15 @@ disable_gravatar_popup = Disable Gravatar and custom sources. All avatars must b
 federated_avatar_lookup = Enable Federated Avatars Lookup
 federated_avatar_lookup_popup = Enable federated avatar lookup using Libravatar.
 disable_registration = Disable Self-registration
-disable_registration_popup = Disable user self-registration, only admin can create accounts.
+disable_registration_popup = Disable self-registration; only admins will be able to create accounts.
```

Comment:
Since this is an ini-file, `;` delimits a comment and the entire string would need to be quoted like so:
```ini
foo = "Bar; baz"
```
Second thought on this is that semi-colon is slightly confusing to non-native speakers. Who either use English out of preference (like myself), or there simply not being a translation for their language yet.

Reply:
Ah, yeah, nice catch on the comment delimiter! Does the particular ini implementation used by Gitea consider those as comments? The Wikipedia page on ini files and most parsers I have used typically only consider them comments if the semicolon is the first character on a line.

```ini
; this would normally be a comment
key = value; but this wouldn't
```

Anyway, I'll double check and adjust those in a little if necessary. Thanks!

---

>Second thought on this is that semi-colon is slightly confusing to non-native speakers.

Maybe you have an alternative suggestion? I can see where you're coming from. But, I just think that for these existing comma splices, if you don't rewrite the actual sentences themselves, semicolons are the best punctuation mark to use as they let you know that these are two sentences that can and do stand on their own but are also heavily related to each other—which is exactly the intended purpose of these misused commas.

Reply:
1) We use `go-ini/ini`, and yes it does. https://github.com/go-ini/ini/#comment
2) Harder to answer. For now I think we can use semicolons since they're not used that often, and people will probably just think it's a type-o :trollface:

Reply:
Alright, I'll go ahead and fix those when I'm home (Damn you, INI!) 😉

Reply:
@bkcsoft Fixed all the issues with the ini file.